### PR TITLE
Notes/GitHub and markdown

### PR DESCRIPTION
**Pull request by KilianBau**
Hope it's fine :)

### CODE REVIEW

**Review by mariusgolabski — COMMENTED** [review]
Yes, that looks good!

File: shell-and-git/PRs-and-Branches.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+1. neuer branch --> git switch -c **name**
+2. zwischen branches wechseln --> git switch **name**
+3. Übersicht lokale branches --> git branch
+4. Übersicht lokale und remote branches --> git branch -a
+5. erstes Mal branch mit Befehl pushen (git push -u origin **name**), danach normal (git push)
+6. branch löschen nach erfolgreichem merchen --> git branch -d **name**
+
+_Skizze_
+
+            O<---O<---O
```
Comment (jonas8900) [inline shell-and-git/PRs-and-Branches.md:12]: üff, this is nice !

**Review by jonas8900 — COMMENTED** [review]
LGTM!

File: shell-and-git/PRs-and-Branches.md (hunk 1 above)
Comment (milodevhh) [reply shell-and-git/PRs-and-Branches.md:12]: Hi guys, well done good to go ;)

**Pull request merged**